dev-system/src/cmr/dev_system/system.clj

```diff
@@ -318,9 +318,7 @@
      :pre-components (u/remove-nil-keys
```

Comment:
Would we want to get rid of the `require` and the `defconfig` that sets the port on line 285?